### ISSUE TITLE
One bin is missing in ODMR scan

### ISSUE THREAD

**Issue by drogenlied** [drogenlied/qudi#57]
**Reported by drogenlied on 28 Oct 2015 18:30 UTC**
Scans with few bins/frequencies are the most affected.

**Comment by drogenlied** [comment]
**Modified by drogenlied on 4 Jan 2016 00:33 UTC**

**Comment by drogenlied** [comment]
**Comment by latchr on 9 May 2016 14:08 UTC**
Milestone renamed

**Comment by drogenlied** [comment]
**Modified by drogenlied on 14 Jun 2016 11:55 UTC**

**Comment by drogenlied** [comment]
**Comment by drogenlied on 21 Jun 2016 10:31 UTC**
Tested SMIQ03, list reset via ABORT or CW/LIST mdoe switch does not seem to work or timings are too short for first bin.

Result: second frequency is in first bin, first frequency is in last bin.

Will also test SMIQ06.

**Comment by drogenlied** [comment]
**Modified by drogenlied on 21 Jun 2016 10:31 UTC**

**Comment by drogenlied** [comment]
**Comment by latchr on 21 Jun 2016 15:41 UTC**
Same on SMIQ06.

**Comment by drogenlied** [comment]
**Comment by drogenlied on 22 Jun 2016 11:43 UTC**
Ticket retargeted after milestone closed

**Comment by drogenlied** [comment]
**Comment by drogenlied on 27 Jun 2016 15:03 UTC**
Fixed for SMIQ.

**Comment by drogenlied** [comment]
**Comment by drogenlied on 28 Jun 2016 12:11 UTC**
Ticket retargeted after milestone closed

**Comment by drogenlied** [comment]
**Comment by GitHub <noreply@github.com> on 5 Jul 2016 11:52 UTC**
In [5d85e59](changeset:"5d85e5904c46d72d77795f90e8186ff082c8b9e2"):
```CommitTicketReference repository="" revision="5d85e5904c46d72d77795f90e8186ff082c8b9e2"
Merge pull request #7 from Ulm-IQO/odmr-bins

All bins present, fixes #33
```
